### DISCUSSION TITLE
help me ,please~

### DISCUSSION THREAD

**Discussion by BoboHeroes** [Chia-Network/bladebit#155] [Q&A] [answered]
I am a computer rookie, but I love this software you made very much. I have always used it on the Chia client, but I want to use it more conveniently. I want to ask how to run it directly under win10, where can I enter my key and pool information, and then draw directly. Thank you very much

**Comment by harold-b** [comment, marked as answer]
Hi, you need to enter your key and pool info via the command line.

You might want to read a little bit about using a command line if you are not familiar with it.

Once you understand it a bit, download the latest release from here https://github.com/Chia-Network/bladebit/releases (or build it from source). Extract bladebit.exe from the compressed archive. Then open a command line in the same directory where you extracted bladebit and run it with the parameters you want.

You can read about the parameters by running bladebit with the `--help` parameter:
```bash
bladebit --help
```

The most useful ones for you will be `-f` and `-c` which specify the farmer public key and pool contract address, respectively. The final argument in the command line will be the destination directory where you want to write your plot file. If don't specify any, the current directory will be used, which is probably not desired.

Example:
```bash
bladebit -f YOUR_FARMER_KEY_HERE -c YOUR_POOL_CONTRACT_ADDRESS_HERE D:\Path\to\store\your\plots
```
